tests/integration/targets/inventory/test.yml

```diff
@@ -0,0 +1,25 @@
+- name: Test inventory plugin
+  hosts: localhost
+  gather_facts: false
+
+  tasks:
+    - ansible.builtin.assert:
```

Comment:
Please split this huge assert list into several smaller asserts with `name:`s so it's more obvious what you're testing.

Reply:
Splited